bgpd/bgp_updgrp_packet.c

```diff
@@ -522,7 +527,12 @@ struct stream *bpacket_reformat_for_peer(struct bpacket *pkt,
 			 * party nexthop is not propagated for EBGP right now.
 			 */
 			switch (nhlen) {
-			case BGP_ATTR_NHLEN_IPV6_GLOBAL:
+
+			case IPV6_MAX_BYTELEN:
```

Comment:
I would leave this as it is:
```
case BGP_ATTR_NHLEN_IPV6_GLOBAL:
case BGP_ATTR_NHLEN_IPV6_LL:
  break;
```

Reply:
This would give compilation error as both the macros gets replaced with the same value.

Reply:
I don't know the right way to resolve this (and the comment does help) but it seems "wrong" to test for IPV6_MAX_BYTELEN since it doesn't convey what you're doing.  This isn't a nak, but it seems a better approach might be to have a definition for "BGP_ATTR_NHLEN_IPV6_GLOBAL_OR_LL" or something like that.  Other ideas welcome and if the current approach doesn't bother anyone else, it's not critical.